IMU1:
 maxX: 0
 minX: 0
 maxY: 0
 minY: 0